pkg/fleetautoscalers/fleetautoscalers_test.go

```diff
@@ -49,10 +49,12 @@ func (t testServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	res, err := ioutil.ReadAll(r.Body)
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
```

Comment:
Wondering if these changes should be pulled into their own PR(s) - they don't seem tied to the vuln issues.

Also curious if they are the cause / exposing the issue of the latest failure, which I haven't seen before:

```
--- FAIL: TestAutoscalerBasicFunctions (54.80s)
    fleetautoscaler_test.go:103: 
        	Error Trace:	fleetautoscaler_test.go:103
        	Error:      	Received unexpected error:
        	            	invalid character '1' after object key:value pair
        	Test:       	TestAutoscalerBasicFunctions
        	Messages:   	could not patch fleetautoscaler
```

Reply:
These changes are required because I need to use `go install lint` instead of curl|bash and `go install` only supports with go version later than 1.16 and go lint 1.43. After using the latest go lint, a few more issues are reported for `make lint` and these changes are needed for test to pass. This specific test error is because lint recommended to change a string format in line 373 in this file. I'm fixing it right now.

Reply:
Aaah! That makes a lot of sense. 👍🏻